src/trpc/client.tsx

```
@@ -24,7 +24,8 @@ function getQueryClient() {
 }
 function getUrl() {
   const base = (() => {
-    return `${process.env.NEXT_APP_URL}`;
+    if (typeof window !== undefined) return "";
```

Biome (2.1.2)

[error] Invalid typeof comparison.

Compare with one of the following string literals:


(lint/correctness/useValidTypeof)